torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1330,6 +1348,25 @@ void ComputeConstant(Node* n, int opset_version) {
       ProcessUnchangeNode(n);
       break;
     }
+    case ::c10::onnx::Squeeze: {
+      if (ConstantValueMap::HasShape(n->input(0)->debugName())) {
+        auto input0_shape_size =
+            ConstantValueMap::GetShape(n->input(0)->debugName())
+                .value()
+                .sizes();
+        if (input0_shape_size.has_value()) {
+          auto input0_shape_value = input0_shape_size.value();
+          if (n->inputs().size() == 1) {
```

Comment:
Add a comment so we understand why we only consider `n->inputs().size() == 1` here:
```// For n->inputs().size() > 1, the shape inference is handled via onnx shape inference. ```